test/basic.html

```diff
@@ -173,6 +202,45 @@
           assert.isTrue(smallRect.height >= 1 * parseFloat(smallStyle.fontSize));
         });
       });
+
+      suite('ink size', function() {
+        var checkboxes;
+
+        setup(function() {
+          checkboxes = fixture('WithDifferentSizes2');
+        });
+
+        test('`--paper-checkbox-ink-size` sets the ink size', function() {
```

Comment:
Can you check the default size (when the custom property is unset) is the number of px we expect today?